src/components/ResultsHeaderContainer/ResultsHeaderContainer.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
+import { QuizResultsConfig } from '../../types';
+
+export interface IResultHeaderContainerProps {
+  resultsConfig?: QuizResultsConfig | null;
+  hasNoResults: boolean;
+}
+
+export default function ResultsHeaderContainer({
+  hasNoResults,
+  resultsConfig,
+}: IResultHeaderContainerProps) {
+  let resultsTitle: string;
+  if (hasNoResults) {
```

Comment:
Can we merge this if else block statement with the one below on line 26 to 34 since they use mostly the same conditions?